challenges/prototypes.js

```diff
@@ -4,28 +4,39 @@
 
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
-*/
+*/ function CuboidMaker(properties) {
+  this.length = properties.name;
+  this.width = properties.width;
+  this.height = properties.height;
+}
 
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
   
   Formula for cuboid volume: length * width * height
 */
-
+  
 
 /* == Step 3: Surface Area Method ==
   Create another method using CuboidMaker's prototype that returns the surface area of a given cuboid's length, width, and height. 
 
   Formula for cuboid surface area of a cube: 2 * (length * width + length * height + width * height)
 */
 
-
+  CuboidMaker.prototype.volume() = function () {
+    return `${this.length} * ${this.width} + *${this.length} * ${this.height}`
```

Comment:
* Consider returning the actual value of the calculation, and not a string of the process. Like so:
```return (this.length * this.width + this.length * this.height)```